homeassistant/components/adax/climate.py

```diff
@@ -164,3 +180,12 @@ async def async_update(self) -> None:
         self._attr_target_temperature = data["target_temperature"]
         self._attr_current_temperature = data["current_temperature"]
         self._attr_available = self._attr_current_temperature is not None
+        if data["target_temperature"] == 0:
+            self._attr_hvac_mode = HVACMode.OFF
+            self._attr_icon = "mdi:radiator-off"
+            if self._attr_target_temperature == 0:
+                self._attr_target_temperature = self.min_temp
+        else:
+            self._attr_hvac_mode = HVACMode.HEAT
+            self._attr_icon = "mdi:radiator"
```

Comment:
```suggestion
```

Reply:
The non-local AdaxDevice longer up in the code still uses attr_icon, look at lines 125-132. Therefore, I have implemented it the same way. I suggest to accept this non icons.json for now for consitency, and then rather move to icons.json for the entire file as an improvement in the future?